packages/privacy-toolset/src/do-not-sell-dialog/do-not-sell-dialog.stories.tsx

```diff
@@ -10,17 +10,41 @@ export default {
 
 export const Default: Story< DoNotSellDialogProps > = ( args ) => {
 	const [ isActive, setActive ] = useState( false );
-	return <DoNotSellDialog { ...args } isActive={ isActive } onToggleActive={ setActive } />;
+	return <DoNotSellDialog { ...args } isActive={ isActive } onToggleActive={ setActive } isOpen />;
 };
 
 Default.args = {
 	content: {
 		title: 'Do Not Sell or Share My Data',
-		longDescription: [
-			'Sed placerat, orci ullamcorper varius interdum, diam lectus vestibulum lacus, at varius nunc ipsum quis orci. Ut dui libero, ornare at luctus ac, bibendum sit amet risus. Donec tempus nisl nec arcu feugiat, sit amet fermentum velit varius. Nam sed malesuada enim, eu lobortis neque. Cras metus diam, sollicitudin non diam eu, bibendum tempor eros. Donec vehicula venenatis nunc semper pharetra. Nunc tristique nisl id tristique dictum.',
-			'Sed auctor est nec luctus egestas. Sed tincidunt auctor fringilla. In eleifend diam sed egestas semper. Donec vitae condimentum urna. Morbi non consectetur purus. Etiam faucibus, nunc id laoreet dapibus, arcu nisi sollicitudin turpis, sit amet iaculis lacus nulla eget magna. Suspendisse sed iaculis erat. Duis sed sem vitae velit pharetra convallis elementum vitae nibh. Proin ornare nunc pellentesque posuere ultrices. Fusce lacinia ligula id condimentum semper. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque viverra fringilla vehicula.',
-			'Aenean euismod, sem id tincidunt aliquet, augue ipsum dignissim dui, finibus pretium tellus massa sit amet ipsum. Aliquam auctor magna et dui feugiat, et imperdiet enim cursus. Donec at nulla id dolor pharetra rutrum. Vivamus ornare, massa ac finibus laoreet, neque turpis vulputate ante, a fringilla felis ante a lectus. Suspendisse pulvinar nisi eget est laoreet, a maximus velit finibus. Ut sed venenatis justo. Proin lobortis feugiat aliquam. Aliquam sem velit, tempus vel scelerisque sit amet, suscipit a tellus. Nullam a tellus eros. Quisque ac fermentum felis. Nam sagittis finibus erat.',
-		],
+		longDescription: (
```

Comment:
I think this works too, and provides the flexibility I was looking for. My initial idea was to pass in this content this way
```tsx
<DoNotSellDialog ...props>
{ long description content goes here }
</DoNotSellDialog>
```

and then that would be defined in the props as just
```tsx
content: {
   title: string,
   toggleLabel: string,
   closeButton: string,
},
onClose: function,
children: ReactNode
```

I don't think this is required as this approach you went with has the benefits I listed initially, just thought I'd lay out my initial idea better here 😄

Reply:
Ah, I see; thanks for the idea! I think I prefer to use it as a prop, as I would imagine children are something that we can fully control, e.g. the full layout of the dialog. This prop is just nested as part of a larger layout, so it might be just a bit confusing. I like the idea of passing ReactNode instead of an array of strings, though!